h2-migration-tool/migration.sh

```diff
@@ -27,14 +27,23 @@ read dest_dir
 
 wget -c https://repo1.maven.org/maven2/com/h2database/h2/2.1.210/h2-2.1.210.jar || exit 1
 wget -c https://repo1.maven.org/maven2/com/h2database/h2/1.4.199/h2-1.4.199.jar || exit 1
+wget -c https://repo1.maven.org/maven2/com/h2database/h2/1.3.175/h2-1.3.175.jar || exit 1
 
-db_files=("$src_dir"/*.mv.db)
+db_files=("$src_dir"/*.mv.db "$src_dir"/*.h2.db)
 for filepath in "${db_files[@]}"; do
-    dbname=$(basename "$filepath" .mv.db)
+    if [[ "$filepath" == *"*.mv.db" || "$filepath" == *"*.h2.db" ]]; then
```

Comment:
Whats the purpose of this check?

Reply:
When there are no files of either .mv.db or .h2.db extensions, a new file will be created with the db name *.mv.db or *.h2.db at https://github.com/wso2/samples-is/pull/347/files#diff-92f24a1047b8e36fd5b8033907c315eeb47d60cacc4900e2242daef939328365R46.